src/Controls/tests/Core.UnitTests/AppThemeTests.cs

```diff
@@ -195,6 +195,36 @@ public void ThemeBindingRemovedOnOneTimeBindablePropertyWhenPropertySet()
 			Assert.Equal(Colors.Pink, shell.FlyoutBackgroundColor);
 		}
 
+		void validateRadioButtonColors(RadioButton button, SolidColorBrush desiredBrush)
+		{
+			if (button.Children[0] is Frame frame)
+			{
+				if (frame.Children[0] is Grid grid)
+				{
+					if (grid.Children[0] is Shapes.Ellipse outerEllipse)
+					{
+						Assert.Equal(desiredBrush, outerEllipse.Stroke);
+					}
+					if (grid.Children[1] is Shapes.Ellipse innerEllipse)
+					{
+						Assert.Equal(desiredBrush, innerEllipse.Fill);
+					}
+				}
+			}
```

Comment:
Simplify the nested ifs:
```suggestion
			if (button.Children[0] is Frame frame && frame.Children[0] is Grid grid))
			{
				if (grid.Children[0] is Shapes.Ellipse outerEllipse)
				{
					Assert.Equal(desiredBrush, outerEllipse.Stroke);
				}
				if (grid.Children[1] is Shapes.Ellipse innerEllipse)
				{
                	Assert.Equal(desiredBrush, innerEllipse.Fill);
				}
			}
```

Reply:
Maybe we should actually not use `is` here in the test. Rather cast and use directly as we want the test to fail if something changes. If for some reason a child is not the expected type then it should fail.